src/parser/jobs/pld/modules/Atonement.tsx

```diff
@@ -34,14 +38,20 @@ export default class Atonement extends Analyser {
 		}, () => this.stacksUsed++)
 
 		this.addEventHook({
-			type: 'statusApply',
-			target: this.parser.actor.id,
-			status: this.data.statuses.SWORD_OATH.id,
-		}, () => this.stacksGained = this.stacksGained + STACKS_GAINED)
+			type: 'combo',
+			source: this.parser.actor.id,
+			action: this.data.actions.ROYAL_AUTHORITY.id,
+		}, this.onGenerateStacks)
 
 		this.addEventHook('complete', this.onComplete)
 	}
 
+	private onGenerateStacks(): void {
+		this.overcap += Math.max(this.currentStacks + STACKS_GAINED - MAX_STACKS, 0) // Tracking overcap in case someone wants to do something with it later
```

Comment:
```suggestion
		// Tracking overcap in case someone wants to do something with it later
		this.overcap += Math.max(this.currentStacks + STACKS_GAINED - MAX_STACKS, 0)
```